config/management/src/smoke_test.rs

```diff
@@ -235,10 +236,10 @@ fn attach_validator_full_node(
     );
 
     let v_vfn_network_address = v_vfn.listen_address.clone();
-    let v_vfn_pub_key = libra_secure_storage::config::identity_key(v_vfn).public_key();
+    let v_vfn_pub_key = identity_key(v_vfn).public_key();
```

Comment:
its not very rusty to import functions, we instead import modules / crates, so if you really wanted to "optimize" the right approach would have been config::identity_key

but that's not very clear so I left in the full path.